python/tests/test_describe.py

```diff
@@ -0,0 +1,273 @@
+from . import merge_files, assert_log
+from dlisio.core import fingerprint as fp
+from dlisio.plumbing import *
+
+from io import StringIO
+
+import dlisio
+import pytest
+
+@pytest.fixture(scope="module")
+def fpath(tmpdir_factory, merge_files):
+    path = str(tmpdir_factory.mktemp('semantic').join('semantic.dlis'))
+    content = [
+        'data/semantic/envelope.dlis.part',
+        'data/semantic/file-header.dlis.part',
+        'data/semantic/origin.dlis.part',
+        'data/semantic/well-reference-point.dlis.part',
+        'data/semantic/axis.dlis.part',
+        'data/semantic/long-name-record.dlis.part',
+        'data/semantic/channel.dlis.part',
+        'data/semantic/frame.dlis.part',
+        'data/semantic/path.dlis.part',
+        'data/semantic/zone.dlis.part',
+        'data/semantic/parameter.dlis.part',
+        'data/semantic/equipment.dlis.part',
+        'data/semantic/tool.dlis.part',
+        'data/semantic/process.dlis.part',
+        'data/semantic/computation.dlis.part',
+        'data/semantic/measurement.dlis.part',
+        'data/semantic/coefficient.dlis.part',
+        'data/semantic/calibration.dlis.part',
+        'data/semantic/group.dlis.part',
+        'data/semantic/splice.dlis.part',
+        'data/semantic/message.dlis.part',
+        'data/semantic/comment.dlis.part',
+        'data/semantic/update.dlis.part',
+        'data/semantic/unknown.dlis.part',
+    ]
+    merge_files(path, content)
+    return path
+
+@pytest.fixture(scope="module")
+def batch(tmpdir_factory, fpath):
+    with dlisio.load(fpath) as f:
+        yield f
+
+
+def getobjects(objs):
```

Comment:
This one has to be updated - hence maintained with every new added object.
I am wondering if it wouldn't be easier to forgo this method and simply test describe on all the objects. I don't think overhead would be really big.

And for the fpath. Again I doubt that we won't forget to update the list here and in semantics.
It would be cool if we could move the files list to init - once place to update looks much better.